dotnet/src/Planners/Planners.Core/Handlebars/skPrompt.handlebars

```diff
@@ -0,0 +1,138 @@
+{{#message role="system"}}## Instructions
```

Comment:
Should the filename be `skprompt.txt`?

Reply:
No, I intentionally made the extension .handlebars to have the formatting benefits in the IDE
![image](https://github.com/microsoft/semantic-kernel/assets/125500434/657db287-2762-4df2-8464-2d9578ed93cf)

Reply:
I believe this is all changing with the new semantic function template format @matthewbolanos is proposing, but in the meantime, I can change it to skprompt.txt for consistency if you think that's best?

Reply:
No, I didn't know that was supported file type. This is very cool.